CHANGES.md

```diff
@@ -2,7 +2,8 @@
 
 ## 1.4.0
 
-This release adds support for Linux s390x.
+This release adds support for Linux s390x
+and fixes the `scie.platform` and `scie.platform.arch` placeholders for armv7l.
```

Comment:
This will need to be its own 1.4.1 release with accompanying Cargo.toml version bump. See RELEASE.md.

Reply:
Ahh did not realized you already had made a new release. I'll fix.